lib/logstorage/block_test.go

```diff
@@ -205,3 +207,73 @@ func TestBlockMustInitFromRows_Overflow(t *testing.T) {
 	f(15, 30, 15)
 	f(maxColumnsPerBlock+1000, 1, maxColumnsPerBlock)
 }
+
+func TestBlockUncompressedSizeBytes(t *testing.T) {
+	f := func(timestamps []int64, rows [][]Field) {
```

Comment:
It looks like the `timestamps` arg isn't needed here, since `_time` field size doesn't depend on the particular timestamp value.